students/17th/team3/hojinjenechoi/user/views.py

```diff
@@ -0,0 +1,55 @@
+import json
+
+from django.views           import View
+from django.http            import JsonResponse 
+
+from user.models  import User
+
+class SignUpView(View):
+    def post(self, request):
+        signup_data = json.loads(request.body)
+
+        try:
+            if '@' not in signup_data['email'] or '.' not in signup_data['email']:
+              return JsonResponse({'message':'INVALID_EMAIL'}, status=400)
+
+            if User.objects.filter(email=signup_data['email']).exists() or User.objects.filter(phone=signup_data['phone']).exists():
+                return JsonResponse({'message':'USER_ALREADY_EXISTS'}, status=409)
+            
+            if User.objects.filter(nickname=signup_data['nickname']).exists():
+                return JsonResponse({'message':'NICKNAME_ALREADY_EXISTS'}, status=409)
+
+            if len(signup_data['password']) < 8:
```

Comment:
```suggestion
            if len(signup_data['password']) < MINIMUM_PASSWORD_LENGTH:
```
8 이라는 비교값 또한 하드코딩이므로~! 상수화 해서 관리 해주세요.